crates/holo_hash/src/hash.rs

```diff
@@ -157,6 +157,16 @@ impl<T: HashType> HoloHash<T> {
         assert_length!(HOLO_HASH_FULL_LEN, &self.hash);
         self.hash
     }
+
+    /// Get the hex representation of the hash bytes
+    pub fn to_hex(&self) -> String {
+        use std::fmt::Write;
+        let mut s = String::with_capacity(self.hash.len());
+        for b in &self.hash {
+            write!(&mut s, "{:02x}", b).ok();
+        }
+        s
+    }
```

Comment:
I'll do that as a quick follow-up, since we already had it reinvented twice before I added a third

Reply:
Actually, the instance of this in `holo_hash` should not be replaced with subtle-encoding, since that would require pulling in that dependency, and `to_hex()` could be something useful in wasm which shouldn't require a whole extra dependency of bloat. So the case for DRYing up the remaining two instances is less compelling.